planning/autoware_obstacle_cruise_planner/include/autoware/obstacle_cruise_planner/common_structs.hpp

```diff
@@ -94,6 +97,8 @@ struct Obstacle
   rclcpp::Time stamp;  // This is not the current stamp, but when the object was observed.
   double ego_to_obstacle_distance;
   double lat_dist_from_obstacle_to_traj;
+  double tangent_velocity;
+  double normal_velocity;
```

Comment:
I feel this name has unclear meaning.
This sign of this value depends the location of the obstacle.
So I suggest to change this variable name, for example, `approach velocity`.